src/node/tests/elder.rs

```diff
@@ -456,67 +447,6 @@ fn add_and_promote_member() {
     assert!(env.is_candidate_elder());
 }
 
-#[test]
-fn remove_member() {
```

Comment:
why these tests are removed?

Reply:
They are being converted to the new style. Currently the new tests are all in `node/tests/mod.rs` but we might want to split them up into multiple modules when there is too many of them.